catalyst/utils/config.py

```diff
@@ -5,11 +5,36 @@
 from pathlib import Path
 import re
 
+from omegaconf import DictConfig, ListConfig
 import yaml
 
 LOG = getLogger(__name__)
 
 
+def omegaconf_to_dict(config):
```

Comment:
Use the supported `OmegaConf.to_container(cfg, resolve=True)` to convert an OmegaConf object to a primitive container (dict or list).
You do not need to implement this function (I am also guessing that this implementation will fail in various cases covered by the official implementation).

Reply:
Cool! I couldn't find this method in docs. Very thanks!

Reply:
[Here](https://omegaconf.readthedocs.io/en/latest/usage.html#omegaconf-to-container).